src/main/webapp/WEB-INF/view/mentions.jsp

```diff
@@ -9,30 +12,32 @@
   <link rel="stylesheet" href="/css/main.css">
 </head>
 <body>
-  <%String name = (String) request.getSession().getAttribute("user");
-    int numUsers = UserStore.getInstance().getSize();
-    int numConvo = ConversationStore.getInstance().getSize();
-    int numChats = MessageStore.getInstance().getSize();
-    boolean admin = name != null && (name.equals("gaurijain") ||
-    name.equals("beckyqiu") || name.equals("mariorios"));%>
+  <%String name = (String) request.getSession().getAttribute("user");%>
   <nav>
     <a id="navTitle" href="/">CodeU Chat App</a>
     <a href="/conversations">Conversations</a>
-    <% if(admin) {%>
-      <a href="/admin.jsp">Admin</a>
-      <% } %> 
-    <% if(request.getSession().getAttribute("user") != null){%>
+    <% if(name != null){%>
       <a>Hello <%=name%>!</a>
       <a href="/mentions">Mentions</a>
     <% } else{ %>
       <a href="/login">Login</a>
     <% } %>
     <a href="/about.jsp">About</a>
+    <% boolean admin = name != null && UserStore.getInstance().isAdminRegistered(name);
+      if(admin) {%>
+      <a href="/admin">Admin</a>
+      <% } %>
   </nav>
   <div id="container">
     <div style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
       <h1>Your Mentions</h1>
-      <p>You will show messages you were mentioned in here! (Not yet!)</p>
+      <p>You will show messages you were mentioned in here! <%
+      if(name!= null){
+        for (Mention x: UserStore.getInstance().getUser(name).getMentions()){
+        %><%=x.getMessage()%> <%=x.getConversation()%>
```

Comment:
Add a new line here? Does "/n" works in this case? I am ok with this quick pull request but if it's a quick fix let's do it here. If it's not, it's fine to leave it for now.

Reply:
Hey Becky, as you can see it was already merged so I didn't have a chance to fix it, but we planned that Ernesto would fix the UI and display mentions on the jsp, so I think he will already find a better way to display it than this!